tasks/gitlab_helpers.py

```diff
@@ -91,7 +105,32 @@ def get_link_to_job_on_main(job_name: str):
     }
     query_string = to_query_string(query_params)
     quoted_query_string = quote(string=query_string, safe="")
-    return f"{CI_VISIBILITY_URL}?query={quoted_query_string}"
+    return f"{PIPELINE_VISIBILITY_URL}?query={quoted_query_string}"
+
+
+def get_link_to_test_runs(job_id: str):
+    query_params = {
+        "test_level": "test",
+        "@ci.job.url": f"\"https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/{job_id}\"",
```

Comment:
Could we use `job.name` and `git.branch` instead? It's not strictly similar if you have several executions on the same branch so maybe the commit id. It's then easier to navigate in ci-vis I guess

Reply:
If you retry a job in Gitlab, it will have a different ID but the branch, commit, and job name will remain the same.

Reply:
CI Test Vis executions miss the `ci.job.id`, I asked for it